<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1719</width>
    <height>1005</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Darwin Studio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/darwin.png</normaloff>:/resources/darwin.png</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::GroupedDragging</set>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QTabWidget" name="sandbox_tabs">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1719</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_evolution">
    <property name="title">
     <string>&amp;Evolution</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_run"/>
    <addaction name="action_pause"/>
    <addaction name="action_reset"/>
    <addaction name="separator"/>
    <addaction name="action_new_experiment"/>
    <addaction name="action_open_experiment"/>
    <addaction name="action_branch_experiment"/>
    <addaction name="action_new_sandbox"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <addaction name="menu_evolution"/>
   <addaction name="menu_window"/>
  </widget>
  <widget class="QToolBar" name="main_toolbar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_run"/>
   <addaction name="action_pause"/>
   <addaction name="action_reset"/>
   <addaction name="separator"/>
   <addaction name="action_new_experiment"/>
   <addaction name="action_open_experiment"/>
   <addaction name="action_branch_experiment"/>
   <addaction name="separator"/>
   <addaction name="action_new_sandbox"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="action_run">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="toolTip">
    <string>Run evolution!</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_pause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause evolution</string>
   </property>
   <property name="shortcut">
    <string>Pause</string>
   </property>
  </action>
  <action name="action_reset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset population</string>
   </property>
  </action>
  <action name="action_open">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Open universe</string>
   </property>
   <property name="toolTip">
    <string>Open an existing universe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_new">
   <property name="text">
    <string>&amp;New universe</string>
   </property>
   <property name="toolTip">
    <string>Create a new universe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_new_sandbox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New Sandbox</string>
   </property>
   <property name="toolTip">
    <string>New Sandbox Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_new_experiment">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New &amp;experiment</string>
   </property>
   <property name="toolTip">
    <string>Create a new experiment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_branch_experiment">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Branch experiment</string>
   </property>
   <property name="toolTip">
    <string>Branch off from an existing experiment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_open_experiment">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open experiment</string>
   </property>
   <property name="toolTip">
    <string>Open a previous experiment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
